app/models/invoice.rb

```diff
@@ -14,4 +15,12 @@ def serialize_order
 
     self.data = Invoice::OrderSerializer.new(order).serializable_hash
   end
+
+  def set_active
+    self.status = 'active'
+  end
```

Comment:
This would override an existing value as well. It will be impossible to set the status to another value, even for test purposes.